lib/model/ForumModel.class.php

```diff
@@ -553,41 +521,40 @@ class ForumDradenVolgenModel extends CachedPersistenceModel {
 
 	protected static $instance;
 
+	protected function maakForumDraadVolgen($draad_id) {
+		$volgen = new ForumDraadVolgen();
+		$volgen->draad_id = $draad_id;
+		$volgen->uid = LoginModel::getUid();
+		$this->create($volgen);
+		return $volgen;
+	}
+
 	public function getAantalVolgenVoorLid() {
 		return $this->count('uid = ?', array(LoginModel::getUid()));
 	}
 
 	public function getVolgersVanDraad(ForumDraad $draad) {
-		return $this->find('draad_id = ?', array($draad->draad_id))->fetchAll(PDO::FETCH_COLUMN, 1);
+		return $this->prefetch('draad_id = ?', array($draad->draad_id))->fetchAll(PDO::FETCH_COLUMN, 1);
```

Comment:
Die hack geeft het eerste field (oftewel uid) terug in een array. Je hebt het al netjes gefixed zag ik. Tnx!